cluster/addons/fluentd-elasticsearch/es-controller.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
     kubernetes.io/cluster-service: "true"
   name: elasticsearch-logging
 spec:
-  replicas: 1
+  replicas: 2
```

Comment:
Since the ingested log information is being stored inside the container's file system rather than on the host filesystem I thought it wise for it to be replicated at least once so if a pod goes down we don't loose any log information.